packages/kit/test/apps/basics/src/hooks.js

```diff
@@ -18,6 +18,20 @@ export const handleError = ({ request, error }) => {
 		: {};
 	errors[request.url.pathname] = error.stack || error.message;
 	fs.writeFileSync('test/errors.json', JSON.stringify(errors));
+	console.error(error.message);
+	if (error.frame) {
+		console.error(error.frame);
+	}
+	if (error.stack) {
+		console.error(error.stack);
+	}
+
+	if (error.message === '"Testing hook exception"') {
```

Comment:
I wonder if the double quotes are needed here?

Reply:
They are needed. Somehow the error that gets thrown in the handle hook is wrapped in double quotes. As my changes did not cause this I thought it was the wanted behaviour and did not look into why it is wrapped in double quotes.